client/src/components/Homepage/style.css

```diff
@@ -0,0 +1,32 @@
+.statusdiv {
+  display: flex;
+  /* margin-left: 10vw; */
+}
+.pendingDiv {
+  display: flex;
+  margin-right: 5%;
+  text-align: center;
+  padding-top: 3vh;
+}
+.closedDiv {
+  display: flex;
+  text-align: center;
+  padding-top: 3vh;
+}
+.selected {
+  background: var(--main-color, blue);
+}
+.status {
+  margin: 2.5vh 1.5vw;
+}
+#circle {
+  margin-top: 2.2vh;
+  margin-left: -0.8vw;
+  margin-right: 1vw;
+  height: 18.8px;
```

Comment:
@Anies12 You might want to round up to 19px here - the decimal pixel height can sometimes cause glitches in certain browsers, if there are things inside this element that are a percentage of height